Source/WebKit/UIProcess/Extensions/Cocoa/_WKWebExtensionDeclarativeNetRequestRule.mm

```diff
@@ -686,7 +709,6 @@ - (void)removeInvalidResourceTypesForKey:(NSString *)ruleConditionKey
     }
 }
 
-// If you are making changes to how Chrome's rules are translated into WebKit rules, increase the hash version number in _WKWebExtensionDeclarativeNetRequestTranslator -- currentDeclarativeNetRequestHashVersion.
```

Comment:
Keep?

Reply:
This is a bit outdated - that variable doesn't exist anymore.

The correct comment is further up in the file:
```
// If any changes are made to the rule translation logic here, make sure to increment currentDeclarativeNetRequestRuleTranslatorVersion in WebExtensionContextCocoa.
```

I need to update that version as well.